rllib/utils/filter.py

```diff
@@ -86,9 +86,15 @@ def __init__(self, shape=None):
 
     def copy(self):
         other = RunningStat()
-        other.num_pushes = self.num_pushes
-        other.mean_array = np.copy(self.mean_array)
-        other.std_array = np.copy(self.std_array)
+        other.num_pushes = self.num_pushes if hasattr(self, "num_pushes") else self._n
+        other.mean_array = (
+            np.copy(self.mean_array)
+            if hasattr(self, "mean_array")
+            else np.copy(self._M)
+        )
+        other.std_array = (
+            np.copy(self.std_array) if hasattr(self, "std_array") else np.copy(self._S)
+        )
```

Comment:
For reference, it was #27864 that made these changes. All changed attribute names seem to be covered here.

Reply:
Nit: A TODO that tells us to remove this additional code at some point. But it does not do any harm here either 😃

Reply:
I can add the nit :D 

I also saw that we get often a [`DeprecationWarning`](https://numpy.org/devdocs/release/1.20.0-notes.html#passing-shape-none-to-functions-with-a-non-optional-shape-argument-is-deprecated) with this class which could be overcome probably by choosing `()` instead of `None` 

I could push that together into one smooth PR if needed

Reply:
Separate PRs for separate problems would be appreciated!